scripts/run_command.sh

```diff
@@ -21,5 +21,8 @@ source "${METAL3_DIR}/lib/network.sh"
 # shellcheck disable=SC1090
 # shellcheck disable=SC1091
 source "${METAL3_DIR}/lib/images.sh"
+# shellcheck disable=SC1090
+# shellcheck disable=SC1091
+source "${METAL3_DIR}/lib/ironic_basic_auth.sh"
```

Comment:
Only basic auth, don't you need `ironic_tls_setup.sh` too?

Reply:
Good suggestion! I check and see that calling again the `ironic_tls_setup.sh` doesn't regenerate the certificates, but can ensure that we can access the CA certificate from the e2e test. We may need it in the future.